src/pages/fileCentricCart/store/cart.js

```diff
@@ -92,9 +96,29 @@ const reducers = {
   deleteFiles: (state, item) => {
     const fileIdsAfterDeletion = filterOutIDs(item.fileIds, state.fileIds);
     localStorage.setItem('CartFileIds', JSON.stringify(fileIdsAfterDeletion));
+    const newSortColumn = '';
+    const newSortDirection = '';
+    // if not all ids have been removed, then only the removed file ids values
+    if (fileIdsAfterDeletion.length > 0) {
+      return {
+        ...state,
+        fileIds: fileIdsAfterDeletion,
+      };
+    }
     return {
       ...state,
       fileIds: fileIdsAfterDeletion,
+      sortColumn: newSortColumn,
+      sortDirection: newSortDirection,
+    };
+  },
+  sortOrder: (state, item) => {
+    const newSortColumn = item.sortColumn;
+    const newSortDirection = item.sortDirection;
+    return {
+      ...state,
+      sortColumn: newSortColumn,
```

Comment:
one last and minor change, you dont need a new variable to be declared in here
sortColumn:  item.sort column,